boards/arm/rddrone_fmuk66/rddrone_fmuk66.dts

```diff
@@ -307,6 +308,22 @@ zephyr_udc0: &usbotg {
 	status = "okay";
 	pinctrl-0 = <&i2c1_default>;
 	pinctrl-names = "default";
+
+	/*
+	 * This board does not have a BME280, it has a BMP280.
+	 * The two parts are incredibly similar, so the BME280 driver should work.
+	 */
+	bme280@76 {
+		compatible = "bosch,bme280";
+		status = "okay";
+		reg = <0x76>;
+	};
+
+	bmm150: bmm150@10 {
+		compatible = "bosch,bmm150";
+		status = "okay";
+		reg = <0x10>;
+	};
```

Comment:
These changes don't belong in a commit that's fixing I2C for the board. Please move to a separate commit

Reply:
@MaureenHelm .. 
Why doesn't startup-delay-us work? Isn't that what it's for?
>> Your comment makes sense, but startup-delay-us did not work for me when I tried a few days ago, but today it is working, I am testing with more thoroughly with startup-delay-us

Reply:
Changed as per your suggestion